malicious-token-contract-ml-py/src/findings.py

```diff
@@ -40,9 +47,16 @@ def malicious_contract_creation(
 
     def safe_contract_creation(
         self,
+        chain_id: int,
         labels: list,
     ) -> Finding:
         self.label = labels
+        self.metadata["anomaly_score"] = calculate_alert_rate(
+            chain_id,
+            BOT_ID,
+            "SAFE-TOKEN-CONTRACT-CREATION",
+            ScanCountType.TRANSFER_COUNT,
```

Comment:
Could you use `ScanCountType.CONTRACT_CREATION_COUNT` for the denominator?
https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#L12